exo-add/features/steps/steps-given.ls

```diff
@@ -17,3 +18,14 @@ module.exports = ->
     @app-dir := path.join process.cwd!, 'tmp'
     fs.empty-dir-sync @app-dir
 
+
+  @Given /^I am in the directory of an application containing a "([^"]*)" service$/, (service-role, done) !->
+    @app-dir := path.join process.cwd!, 'tmp', 'app'
+    @create-empty-app('app', done)
+      .then ~>
```

Comment:
Ah, the promise was in there matching the implementation of the `I am in the directory of an empty app` step, I've fixed it so that neither of them, use promises.